src/coreclr/jit/loopcloning.cpp

```diff
@@ -436,16 +441,20 @@ void LoopCloneContext::PrintBlockConditions(unsigned loopNum)
 //      all be "AND"ed, so statically we know we will never take the fast path.
 //
 //      Sometimes we simply can't say statically whether "V02 > V01.length" is true or false.
-//      In that case, the "pAllTrue" will be false because this condition doesn't evaluate to "true" and
-//      "pAnyFalse" could be false if no other condition statically evaluates to "false".
+//      In that case, `*pAllTrue` will be false because this condition doesn't evaluate to "true" and
+//      `*pAnyFalse` could be false if no other condition statically evaluates to "false".
+//
+//      If `*pAnyFalse` is true, we set that and return, and `*pAllTrue` is not accurate, since the loop cloning
+//      needs to be aborted.
+//
 void LoopCloneContext::EvaluateConditions(unsigned loopNum, bool* pAllTrue, bool* pAnyFalse DEBUGARG(bool verbose))
 {
     bool allTrue  = true;
     bool anyFalse = false;
 
     JitExpandArrayStack<LC_Condition>& conds = *conditions[loopNum];
 
-    JITDUMP("Evaluating %d loop cloning conditions for loop %d\n", conds.Size(), loopNum);
+    JITDUMP("Evaluating %d loop cloning conditions for loop L%02u\n", conds.Size(), loopNum);
```

Comment:
I see `FMT_LP` out there, knocking at the door.

Reply:
+1

Reply:
I almost did this, but deferred it. Will add it to the "to do" list.